gstd/src/msg/encoded.rs

```diff
@@ -140,9 +144,10 @@ pub fn reply_on_stack<E: Encode + MaxEncodedLen>(payload: E, value: u128) -> Res
             const ERROR_LOG: &str = "Unexpected encoding behavior: too large input bytes size";
             let end_offset = self.offset.checked_add(bytes.len()).expect(ERROR_LOG);
             if end_offset > self.buffer.len() {
-                panic!("{}", ERROR_LOG);
+                panic!("{ERROR_LOG}");
             }
-            self.buffer[self.offset..end_offset].copy_from_slice(bytes);
+            let this = &mut self.buffer[self.offset..end_offset];
+            this.copy_from_slice(unsafe { transmute(bytes) });
```

Comment:
btw can we even more optimise it by avoiding copying with some other approach or adding additional methods in codec?

Reply:
We have uninitialized memory where we will write data when encoding. It requires copying bytes anyway.

Reply:
but atm we copy them twice: once for writable buffer, secondly for uninitialised memory